frontends/common/options.cpp

```diff
@@ -97,6 +97,11 @@ CompilerOptions::CompilerOptions() : Util::Options(defaultMessage) {
     registerOption("--toJSON", "file",
                    [this](const char* arg) { dumpJsonFile = arg; return true; },
                    "Dump the compiler IR after the midend as JSON in the specified file.");
+    registerOption("--p4runtime-files", "filelist",
+                   [this](const char* arg) { p4RuntimeFiles = arg; return true; },
+                   "Write the P4Runtime control plane API description to the specified\n"
+                   "files (comma-separated list).  The format is inferred from the file\n"
+                   "suffix: .info, .json, .p4info");
```

Comment:
I don't understand these suffixes apart from the .json one
The one for binary should probably be .bin (if you want to accept .p4info for binary in addition to .bin, that could be a good idea), the one for text should probably be .txt.